scripts/tags/exturl.js

```diff
@@ -4,8 +4,11 @@
 
 'use strict';
 
-var util = require('hexo-util');
+/*jshint camelcase: false */
+var util = require(hexo.base_dir + 'node_modules/hexo/node_modules/hexo-util');
```

Comment:
Try to reproducte it:
```
npm -v
1.3.6
```
```
npm install hexo-cli -g
hexo init blog
```
```
[root@almostover hexo-test]# ./info.sh
Node.js version: v0.10.48
NPM version: 1.3.6
hexo-util module in hexo sub-direcotry:
  ls: cannot access node_modules/hexo/node_modules/hexo-util: No such file or directory
hexo-util nodele in root sub-directory:
  ls: cannot access node_modules/hexo-util: No such file or directory
[root@almostover hexo-test]# cd blog
[root@almostover blog]# .././info.sh
Node.js version: v0.10.48
NPM version: 1.3.6
hexo-util module in hexo sub-direcotry:
  drwxr-xr-x 5 root root 4096 Mar 16 00:11 node_modules/hexo/node_modules/hexo-util
hexo-util nodele in root sub-directory:
  ls: cannot access node_modules/hexo-util: No such file or directory
```
![image](https://cloud.githubusercontent.com/assets/16944225/23971029/44e776e2-09dd-11e7-8bd4-771bb8af62eb.png)